docs/source/installation/methods.rst

```diff
@@ -0,0 +1,67 @@
+Installation methods
+====================
+
+The following are all of the available options for the installing Ceph through different channels.
+We support 3 main installation methods, all managed by the ``ceph_origin`` variable:
+
+- ``repository``: means that you will get ceph installed through a new repository. Later below choose between 'community', 'rhcs' or 'dev'.
+- ``distro``: means that no separate repo file will be added and you will get whatever version of Ceph is included in your Linux distro.
+- ``local``: means that the ceph binaries will be copied over from the local machine (not well tested, use at your own risk)
+
+Origin: Repository
+--------------------
+
+If ``ceph_origin`` is set to ``repository``, you now have the choice between a couple of repositories controlled by the ``ceph_repository`` option:
+
+- ``community``: fetches packages from http://download.ceph.com, the official community Ceph repositories
+- ``rhcs``: means you are a Red Hat customer, additionally you will have tAo select a repository type through ``ceph_repository_type`` (``cdn`` or ``iso``)
+- ``dev``: fetches packages from shaman, a gitbuilder based package system
+- ``uca``: fetches packages from Ubuntu Cloud Archive
+- ``custom``: fetches packages from a specific repository
+
+Community repository
+~~~~~~~~~~~~~~~~~~~~
+
+If ``ceph_repository`` is set to ``community``, packages you will be by default installed from http://download.ceph.com, this can be changed by tweaking ``ceph_mirror``.
+Final step is to select which Ceph release you want to install, ceph-ansible always defaults to the latest stable release (e.g: luminous). So set ``ceph_stable_release`` accordingly.
```

Comment:
we don't default to the latest stable for ``ceph_stable_release`` anymore do we?

Reply:
Right now it's set as ``ceph_stable_release: dummy``.  https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-defaults/defaults/main.yml#L106

Reply:
Oops, you're right, I just forgot my own recent change ^^

Reply:
I'd be +1 on making it ``ceph_stable_release: luminous`` for the 3.0.0 release though.